smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/http-uri-greedy-lables-validator1.smithy

```diff
@@ -0,0 +1,28 @@
+$version: "2"
+namespace smithy.example
+
+service SmithyExample {
+    operations: [
+        OperationOne
+    ]
+}
+
+@http(code: 200, method: "GET", uri: "/example/{greedyOne+}/bar/{greedyTwo+}")
```

Comment:
Should `greedyOne+` emit a "greedy label should be the last label" event?

Reply:
It doesn't as the logic is written, 

```java
if (segments.get(j).isGreedyLabel()) {
    events.add(danger(shape, trait,
                      "At most one greedy label segment may exist in a pattern: " + pattern));
} else if (segments.get(j).isLabel()) {
    events.add(danger(shape, trait,
                      "A greedy label must be the last label in its pattern: " + pattern));
}
```
It could if we convert the `else if` into an `if` by itself, but that will emit two events for this case, I feel like one would be better as the second is *implied* by the first.

Reply:
IMO I think it’ll help to over-emit events since these are events that could be suppressed.

The best case is to force users to explicitly suppress events per extra greedy label and per greedy label that’s not last, but here it’s “blanketed” into one event.